package.json

```diff
@@ -20,10 +20,16 @@
     "ascii-json": "^0.2.0",
     "babel-core": "^6.10.4",
     "babel-loader": "^6.2.4",
+    "babel-plugin-transform-decorators-legacy": "^1.3.4",
+    "babel-plugin-typecheck": "^3.9.0",
```

Comment:
Alright, will check whether deployment to Heroku works without them (Heroku is by default only installing the non-dev dependencies). I just put them there because they are also in the `.babelrc` config and deployment didn’t work without the preset modules at least.